### ISSUE TITLE
Export `useGtag` function

### ISSUE THREAD

**Issue by StijnMaenhautCZ** [johannschopplich/nuxt-gtag#94]
### Describe the feature

Currently, we're importing the `useGtag` function as follows:

```ts
import { useGtag } from '#imports';
```

As we put all our components in Storybook as well, and Storybook can't really cope with `#imports` (it errors out), I was wondering if you could make the `useGtag` function available through the package exports itself?

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Can you think of other implementations of this feature?

### Final checks

- [X] Check existing [issues](https://github.com/johannschopplich/nuxt-gtag/issues).

**Comment by StijnMaenhautCZ** [comment]
Yes, I'm using the [@nuxtjs/storybook](https://www.npmjs.com/package/@nuxtjs/storybook) module. The error is the following:

<img width="553" alt="SCR-20241128-prdy" src="https://github.com/user-attachments/assets/752c7643-79b3-4674-8611-2f8fb4010edb">

Would it be a solution to provide the Nuxt context to `useGtag` as an argument?

**Comment by johannschopplich** [comment]
I don't know how the Nuxt context is set up within Storybook. Do other auto-imports work? Can you remove the explicit import from `#imports` and rely on auto-imports instead?

**Comment by StijnMaenhautCZ** [comment]
Thanks for your quick replies! 

I have been able to bypass all other `#imports` by using a "direct import". This is the only package left 😃 

I have removed the import line, but the issue remains.
